spec/features/visits/03_users_can_edit_a_visit_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "users can edit visits within a vacation" do
+  scenario "signed in user can edit a visit they created" do
+    visit "/"
+    click_link "Sign in with Google"
+    park = FactoryGirl.create(:park, name: "Kenai Fjords")
```

Comment:
park is needed to be associated with a visit. I have moved it into a let statement above.